packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -0,0 +1,106 @@
+// @ts-check
+import React, { useState, useEffect } from 'react';
+import { useOvermind } from 'app/overmind';
+import RightIcon from 'react-icons/lib/md/keyboard-arrow-right';
+import LeftIcon from 'react-icons/lib/md/keyboard-arrow-left';
+import { withRouter } from 'react-router-dom';
+import { Navigation } from 'app/pages/common/Navigation';
+import { SignInButton } from 'app/pages/common/SignInButton';
+import { client } from 'app/graphql/client';
+
+import SidebarContents from './Sidebar';
+import Content from './Content';
+import {
+  Container,
+  Centered,
+  Content as ContentContainer,
+  LoggedInContainer,
+  LoggedInTitle,
+  Sidebar,
+  NavigationContainer,
+  ShowSidebarButton,
+  OffsettedLogo,
+} from './elements';
+
+const Dashboard = props => {
+  const [showSidebar, setShowSidebar] = useState(false);
+
+  const {
+    state: { hasLogIn },
+    actions: { dashboard },
+  } = useOvermind();
+
+  useEffect(() => {
+    dashboard.dashboardMounted();
+    return () => {
+      // Reset store so new visits get fresh data
+      client.resetStore();
+    };
+  }, [dashboard]);
+
+  const onRouteChange = () => {
+    setShowSidebar(false);
+  };
+
+  const toggleSidebar = () => {
+    setShowSidebar(!showSidebar);
+  };
+
+  const { history } = props;
+
+  history.listen(({ state }) => {
+    if (!!state && state.from === 'sandboxSearchFocused') {
+      return;
+    }
+
+    onRouteChange();
+  });
+
+  let DashboardContent = (
+    <>
+      <Sidebar active={showSidebar}>
+        <SidebarContents />
+        <ShowSidebarButton onClick={toggleSidebar}>
+          {showSidebar ? (
+            <LeftIcon style={{ color: 'white' }} />
+          ) : (
+            <RightIcon style={{ color: 'white' }} />
+          )}
+        </ShowSidebarButton>
+      </Sidebar>
+
+      <ContentContainer>
+        <Content />
+      </ContentContainer>
+    </>
+  );
+
+  if (!hasLogIn) {
+    DashboardContent = (
+      <Container>
+        <Centered>
+          <LoggedInContainer>
+            <OffsettedLogo />
+            <LoggedInTitle>Sign in to CodeSandbox</LoggedInTitle>
+
+            <SignInButton big style={{ fontSize: '1rem' }} />
```

Comment:
Second typecheck error was already present in unmodified file
![image](https://user-images.githubusercontent.com/3725386/67151168-0d839e80-f2db-11e9-9f3f-ca3766bc265c.png)

Reply:
I can fix this by adding `big?: boolean;` in https://github.com/codesandbox/codesandbox-client/blob/a62cec439d1f80d991ad548b25a62efee9cb0da7/packages/common/src/components/Button/index.tsx#L4

Reply:
Not sure whether I should do that or not.

Reply:
Fixed